tools/azure-plugin/generator/TestProjects/Local/Unbranded-TypeSpec/Unbranded-TypeSpec.tsp

```diff
@@ -0,0 +1,368 @@
+import "@typespec/rest";
```

Comment:
Lets name this Basic-TypeSpec as it won't be unbranded in this repo :-)

Reply:
renamed